src/Sylius/Behat/Page/Checkout/CheckoutSecurityStep.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Checkout;
+
+use Sylius\Behat\SymfonyPageObjectExtension\PageObject\SymfonyPage;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CheckoutSecurityStep extends SymfonyPage
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_checkout_security';
+    }
+
+    /**
+     * @param string $login
+     * @param string $password
+     */
+    public function logInExistingUser($login, $password)
```

Comment:
**weird**? right in the feels :(
I tried to keep sylius security page naming:
![zrzut ekranu 2016-02-09 o 15 48 14](https://cloud.githubusercontent.com/assets/6212718/12919495/990549d6-cf44-11e5-99af-a52065ee8338.png)
So we can log in as guest, log in as existing customer or create new account... I guess method can be named ``logInExistingCustomer`` if you mean that, if not, I must disagree :smile: